tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -47,7 +47,8 @@
 }
 
 DEFAULT_PREFS = {
-    'webapps.update.enabled': False  # disable web apps update
+    'webapps.update.enabled': False,  # disable web apps update
+    'ui.caretBlinkTime': 0
```

Comment:
Perhaps add a comment that says: Make caret permanently visible so imagecompare screenshots are consistent